web/src/compose_closed_ui.ts

```diff
@@ -157,10 +170,25 @@ export function set_standard_text_for_reply_button(): void {
 
 export function update_reply_recipient_label(message?: ComposeClosedMessage): void {
     const recipient_label = get_recipient_label(message);
-    if (recipient_label) {
-        set_reply_button_label(
-            $t({defaultMessage: "Message {recipient_label}"}, {recipient_label}),
-        );
+    if (recipient_label !== undefined) {
+        if (!recipient_label.has_empty_string_topic) {
+            const recipient_label_text = recipient_label.label_text;
+            set_reply_button_label(
+                $t({defaultMessage: "Message {recipient_label_text}"}, {recipient_label_text}),
+            );
+        } else {
+            const topic_display_name = util.get_final_topic_display_name("");
+            const recipient_label_html = $t_html(
+                {
+                    defaultMessage: "Message <z-recipient-label></z-recipient-label>",
```

Comment:
I think ideally we'd arrange for this translation to match the other `Message` one just above to save translation effort, which probably means just reworking the `recipient_label_text` case to use the same string.

Reply:
#33054